sdk/src/logs/batch_log_processor.cc

```diff
@@ -41,6 +41,14 @@ BatchLogProcessor::BatchLogProcessor(std::unique_ptr<LogExporter> &&exporter,
   synchronization_data_->is_force_flush_pending.store(false);
   synchronization_data_->is_force_flush_notified.store(false);
   synchronization_data_->is_shutdown.store(false);
+#  ifdef ENABLE_ASYNC_EXPORT
+  export_data_storage_->export_ids_flag.reserve(max_export_async_);
```

Comment:
Is `export_data_storage_->export_ids_flag.resize(max_export_async_, true);` better?

Reply:
Agreed.